u19_pipeline/recording.py

```diff
@@ -1,204 +1,253 @@
+        print(command)
+        p = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        p.wait()
+        print('aftercommand before comm')
+        stdout, stderr = p.communicate()
+        print('aftercommand after comm')
+        print(stdout.decode('UTF-8'))
+        print(stderr.decode('UTF-8'))
 
+    class File(dj.Part):
+        definition = """
+        # list of files per FOV
+        -> master
+        file_number          : int
+        ---
+        fov_filename         : varchar(255)                 # file name of the new fov tiff file
+        file_frame_range     : blob                         # [first last] frame indices in this file, with respect to the whole imaging session
+        """
 
-@schema
-class Recording(dj.Manual):
-     definition = """
-     recording_id:                      INT(11) AUTO_INCREMENT                  # Unique number assigned to each recording   
-     -----
-     -> RecordingModality    
-     -> lab.Location                        
-     -> StatusRecordingDefinition                                               # current status for recording in the pipeline
-     -> PreprocessParamSet.proj(def_preprocess_paramset_idx='preprocess_paramset_idx')  # reference to params to default preprocess recording (possible to inherit to recordigprocess)
-     -> ProcessParamSet.proj(def_process_paramset_idx='process_paramset_idx')   # reference to params to default process recording  (possible to inherit to recordigprocess)
-     task_copy_id_pni=null:             UUID                                    # id for globus transfer task raw file local->cup
-     inherit_params_recording=1:        boolean                                 # all RecordingProcess from a recording will have same paramSets
-     recording_directory:               varchar(255)                            # relative directory where the recording will be stored on cup
-     local_directory:                   varchar(255)                            # local directory where the recording is stored on system
-     """    
-    
-     class BehaviorSession(dj.Part):
-         definition = """
-         -> master
-         ---
-         -> acquisition.Session
-         """
-    
-     class RecordingSession(dj.Part):
-         definition = """
-         -> master
-         ---
-         -> subject.Subject
-         recording_datetime: datetime
-         """
 
+# Declare recording processing tables --------------------------------------------------
+ephys_element_schema = dj.create_virtual_module(dj.config['custom']['database.prefix'] + 'ephys_element',dj.config['custom']['database.prefix'] + 'ephys_element')
+
+imaging_element_schema = dj.create_virtual_module(dj.config['custom']['database.prefix'] + 'imaging_element',dj.config['custom']['database.prefix'] + 'imaging_element')
 
 @schema
-class StatusProcessDefinition(dj.Lookup):
+class ProcessingStatusDefinition(dj.Lookup):
```

Comment:
From this table in a new schema recording_processing or something like that to avoid circular imports

Reply:
Added these tables to the `recording_process` module/schema.